pipeline/terraform/stack/service_work_batcher.tf

```diff
@@ -1,8 +1,25 @@
 locals {
-  wait_minutes = var.reindexing_state.scale_up_tasks ? 45 : 1
+  # The purpose of the batcher is to minimise the number of times
+  # an individual record is processed, by batching them together when they
+  # are expected to modify each other.  This means that choosing parallelism
+  # and bundle size is not about maximising throughput, but about ensuring
+  # that the largest number of records are processed in the smallest number
+  # of executions.
+
+  # During a reindex, the batcher is expected to receive
+  # roughly 0.5 million messages in roughly 2 hours.
+  # At peak, messages appear at a rate of about 6000 per minute, so
+  # waiting for 20 minutes would perfectly align with processing 120000 at a time.
```

Comment:
I have a vague memory that 120K is an SQS-imposed limit – can you link to some docs that confirm that?